src/main/java/webDriver/GooglePage.java

```diff
@@ -0,0 +1,33 @@
+package webDriver;
+
+import org.openqa.selenium.*;
+
+public class GooglePage {
+    private By searchField = By.id("lst-ib");
```

Comment:
use @FindBy annotation and `WebElement` as type! This is a default PageObject pattern behavior.